src/app/modules/main/wallet_section/module.nim

```diff
@@ -137,6 +137,7 @@ method notifyFilterChanged(self: Module) =
   self.transactionsModule.filterChanged(self.filter.addresses, self.filter.chainIds)
   self.accountsModule.filterChanged(self.filter.addresses, self.filter.chainIds, self.filter.excludeWatchOnly)
   self.sendModule.filterChanged(self.filter.addresses, self.filter.chainIds)
+  self.view.filterChanged(self.filter.addresses[0], self.filter.excludeWatchOnly, self.filter.allAddresses)
```

Comment:
I know it's temporary but it's worth checking if `addresses` is not empty

Reply:
Not temporary on this one :)